docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+launch files inside the `husky_navigation` package:
+
+- `launch/exploration.launch`
+- `launch/exploration_demo.launch`
+
+Remove the commented-out sections so that the files look like this:
+
+**exploration.launch**:
+
+```markup
+<launch>
+
+  <node pkg="frontier_exploration" type="explore_client" name="explore_client" output="screen"/>
+  <node pkg="frontier_exploration" type="explore_server" name="explore_server" output="screen">
+    <param name="frequency" value="1.0"/>
+
+    <!-- Should be less than sensor range -->
+    <param name="goal_aliasing" value="2.0"/>
+    <rosparam file="$(find husky_navigation)/config/costmap_common.yaml" command="load" ns="explore_costmap" />
+    <rosparam file="$(find husky_navigation)/config/costmap_exploration.yaml" command="load" ns="explore_costmap" />
+  </node>
+</launch>
+```
+
+**exploration_demo.launch**:
+
+```markup
+<launch>
+  <!--- Run gmapping -->
+  <include file="$(find husky_navigation)/launch/gmapping.launch" />
+
+  <!--- Run Move Base -->
+  <include file="$(find husky_navigation)/launch/move_base.launch" />
+
+  <!-- Run Frontier Exploration -->
+  <include file="$(find husky_navigation)/launch/exploration.launch" />
+</launch>
+```
+
+#### Running the demo
+
+:::note
+In each terminal window, make sure to source the catkin workspace where you built `frontier_exploration`.
+:::
+
+1. Ensure that the Husky navigation demo package is installed:
+
+    ```bash
+    sudo apt-get install ros-noetic-husky-navigation
+    ```
+
+2. Open window #1 and start the Clearpath-configured Husky simulation environment:
+
+    ```bash
+    roslaunch husky_gazebo husky_playpen.launch
+    ```
+
+3. Open window #2 and start the Clearpath-configured [RViz](http://wiki.ros.org/rviz) visualizer:
+
+    ```bash
+    roslaunch husky_viz view_robot.launch
+    ```
+
+4. Open window #3 and start the move_base demo:
+
+    ```bash
+    roslaunch husky_navigation exploration_demo.launch
+    ```
+
+5. In the RViz visualizer, make sure the visualizers in the Navigation group are enabled.
+
+6. Use the Point tool in the top toolbar to draw a closed polygon on the map that the
+   Husky should explore. Watch the terminal window for instructions.
+
+7. As the robot moves, you should see the grey static map (map topic) grow. Occasionally,
+   the gmapping algorithm will relocalize the robot, causing a discrete jump in the map->odom transform.
+
+8. When the exploration goal is complete, you will see a feedback message in the terminal
+   window. You can now issue a new exploration goal if you wish.
+
+9. To save the generated map, you can run the map_saver utility:
+
+    ```bash
+    rosrun map_server map_saver -f <filename>
+    ```
+
+---
+
+## Testing Husky     {#testing-husky}
+
+Husky robots come preinstalled with a set of test scripts as part of the ``husky_tests`` ROS package, which can be run to verify robot functionality at the component and system levels. 
```

Comment:
change to single `husky_tests`   (just for text consistency)

Reply:
Already fixed in my view